action: broken
packages:
- linux-64/conda-build-3.28.3-py38h578d9bd_0.conda
- linux-64/conda-build-3.28.3-py39h4162558_0.conda
- linux-64/conda-build-3.28.3-py39hf3d152e_0.conda
- linux-64/conda-build-3.28.3-py310hff52083_0.conda
- linux-64/conda-build-3.28.3-py311h38be061_0.conda
- linux-aarch64/conda-build-3.28.3-py38he3eb160_0.conda
- linux-aarch64/conda-build-3.28.3-py39h387a52e_0.conda
- linux-aarch64/conda-build-3.28.3-py39h4420490_0.conda
- linux-aarch64/conda-build-3.28.3-py310h4c7bcd0_0.conda
- linux-aarch64/conda-build-3.28.3-py311hec3470c_0.conda
- linux-ppc64le/conda-build-3.28.3-py38h8328f6c_0.conda
- linux-ppc64le/conda-build-3.28.3-py39h0b1cf3c_0.conda
- linux-ppc64le/conda-build-3.28.3-py39h8d2a46d_0.conda
- linux-ppc64le/conda-build-3.28.3-py310h194a6c8_0.conda
- linux-ppc64le/conda-build-3.28.3-py311h1af927a_0.conda
- osx-64/conda-build-3.28.3-py38h50d1736_0.conda
- osx-64/conda-build-3.28.3-py39h6e9494a_0.conda
- osx-64/conda-build-3.28.3-py39h43b90dd_0.conda
- osx-64/conda-build-3.28.3-py310h2ec42d9_0.conda
- osx-64/conda-build-3.28.3-py311h6eed73b_0.conda
- osx-arm64/conda-build-3.28.3-py38h10201cd_0.conda
- osx-arm64/conda-build-3.28.3-py39h2804cbe_0.conda
- osx-arm64/conda-build-3.28.3-py310hbe9552e_0.conda
- osx-arm64/conda-build-3.28.3-py311h267d04e_0.conda
- win-64/conda-build-3.28.3-py38haa244fe_0.conda
- win-64/conda-build-3.28.3-py39h0d475fb_0.conda
- win-64/conda-build-3.28.3-py39hcbf5309_0.conda
- win-64/conda-build-3.28.3-py310h5588dad_0.conda
- win-64/conda-build-3.28.3-py311h1ea47a8_0.conda
